packages/compass-components/src/hooks/use-confirmation.tsx

```diff
@@ -176,7 +175,6 @@ export const ConfirmationModalArea: React.FC = ({ children }) => {
           className: confirmationProps.hideCancelButton
             ? hideButtonStyles
             : undefined,
-          children: confirmationProps.cancelButtonText ?? 'Cancel',
```

Comment:
button text is always Cancel and LG removed this option. https://www.mongodb.design/component/confirmation-modal/code-docs